crates/sp-domains/Cargo.toml

```diff
@@ -50,6 +50,14 @@ rlp = { workspace = true, optional = true }
 num-traits.workspace = true
 rand = { workspace = true, features = ["min_const_gen"] }
 
+# test-ethereum dependencies
+ethereum.workspace = true
```

Comment:
Why do we need this to be duplicated in test deps and actual deps ?

Reply:
Because it is required by the tests, but optional for production code.